Makefile

```diff
@@ -322,20 +324,20 @@ packagedir: clean embed-download macos-release-binary packaging/vfkit-$(GOARCH).
 	mkdir -p packaging/root/Applications
 	tar -C packaging/root/Applications -xvzf $(TRAY_RELEASE)
 	rm $(TRAY_RELEASE)
-	mv packaging/root/Applications/crc-tray-darwin-$(TRAY_ARCH)/crc-tray.app packaging/root/Applications/Red\ Hat\ OpenShift\ Local.app
-	rm -fr packaging/root/Applications/crc-tray-darwin-$(TRAY_ARCH)
+	mv packaging/root/Applications/crc-tray-darwin-universal/crc-tray.app packaging/root/Applications/Red\ Hat\ OpenShift\ Local.app
+	rm -fr packaging/root/Applications/crc-tray-darwin-universal
```

Comment:
This needs universal tray binaries, and universal vfkit binaries, I don't know if these are ready yet?

Reply:
They are not released in the github releases, for tray we have a PR which generates universal artifact, for admin-helper and vfkit have to manually build and upload the binaries to github releases

Reply:
@anjannath this commit still using the `universal`, shouldn't this removed?

Reply:
This should be fine, its ony the dir that is named `out/macos-universal` and the tray directory is also after extracting the tray archive, the released executables/archives don't contain the `-universal`